giskard_vision/landmark_detection/detectors/metadata_detector.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+
+from giskard_vision.core.detectors.metadata_scan_detector import MetaDataScanDetector
+from giskard_vision.landmark_detection.detectors.surrogate_functions import (
+    relative_volume_convex_hull,
+)
+from giskard_vision.landmark_detection.tests.performance import NMEMean
+from giskard_vision.landmark_detection.types import PredictionResult
+
+from ...core.detectors.decorator import maybe_detector
+
+
+@dataclass
+class NMEMeanMetric(NMEMean):
+
+    @staticmethod
+    def get(pred, truth):
+        return NMEMean.get(PredictionResult(prediction=pred[None, :]), truth[None, :])
```

Comment:
I think there's still a misunderstanding about this,

I don't see the need to override NMEMean like that at all

Reply:
Because it depends on PredictionResult and the format of prediction (here for instance there is [None, :]), so it's hard to factorize in core